gnomad/utils/constraint.py

```diff
@@ -599,14 +599,13 @@ def build_plateau_models(
         1],[1,1,1]]`. Default is None.
     :param weighted: Whether to generalize the model to weighted least squares using
         'possible_variants'. Default is False.
-    :return: A dictionary of intercepts and slopes of plateau models for all sites and
-        populations (optional). The keys for this dictionary are 'total' and 'pop'
-        (optional). The values for 'total' is a dictionary (e.g., <DictExpression of
-        type dict<bool,array<float64>>>), and the value for 'pop' is a nested list of
-        dictionaries (e.g., <ArrayExpression of type array<array<dict<bool,
-        array<float64>>>>>). The key of the dictionary in the nested list is CpG status
-        (BooleanExpression), and the value is an ArrayExpression containing intercept
-        and slope values.
+    :return: A dictionary of intercepts and slopes of plateau models. The keys are
```

Comment:
We may still want to specify `total` and `pop` are keys in this dict. I rephrase it a bit but feel free to change!

Reply:
this looks good to me